.github/workflows/compare_latest_version.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+import os
+import re
+from pathlib import Path
+from urllib.parse import urljoin
+
+import requests
+from bs4 import BeautifulSoup
+
+SNAPHU_URL = "https://web.stanford.edu/group/radar/softwareandlinks/sw/snaphu/"
+
+
+def fetch_stanford_version_url() -> str:
```

Comment:
This function returns two strings

```suggestion
def fetch_stanford_version_url() -> tuple[str, str]:
```